scipy/stats/_stats_py.py

```diff
@@ -3735,6 +3735,14 @@ def f_oneway(*samples, axis=0):
     axis : int, optional
         Axis of the input arrays along which the test is applied.
         Default is 0.
+    equal_var: bool, optional
+        If True (default), perform a standard one-way ANOVA test that
+        assumes equal population variances [2]_.
+        If False, perform Welch's ANOVA test, which does not assume
+        equal population variances [4]_. Modified F-statistics and
+        dof will be calculated by each samples [5]_
```

Comment:
This suggested that the F-statistic calculation is performed according to [5], and perhaps the p-value is determined according to [4]? I don't really understand the need for both references.